Package.swift

```diff
@@ -12,7 +12,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(name: "FinniversKit", url: "https://github.com/finn-no/FinniversKit.git", .exact("91.0.0"))
```

Comment:
This should point to the new tag. Don't forget to run `swift package resolve` from the project root, in case Xcode doesn't do that for you.